_posts/2018-01-24-browser-sizes.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: How to make a chart of your users' window sizes
+author: Stephanie Hobson
+excerpt_separator: <!--more-->
+---
+
+In preparation for the MDN redesign I examined our analytics to get an idea
+of how wide our users' browser windows were. I wanted window widths not screen
+sizes and I thought a chart would tell a more compelling story than a table.
+
+<!--more-->
+
+Here's the chart I made:
+
+![Chart of MDN window widths showing spikes at 1350 and 1900 pixels and very
+little between 420 and 930 pixels.]({{ site.baseurl }}/public/images/2017-07-10-browser-sizes/browser-sizes.png)
```

Comment:
Does this image URL need updating? It's 404 for me (I'm guessing because the post date is different to here).